src/LottoGame.js

```diff
@@ -0,0 +1,130 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { MESSAGE } from './constants/message.js'
+import validate from './utils/validate.js'
+
+class LottoGame {
+	#lottoAmount //로또 개수
+
+	#lottoRandomNumberArr = [] //로또 랜덤값 배열 (이중 배열)
+
+	#winnerNumberArr //당첨 번호 배열
+
+	#bonusNumber //보너스 번호
+
+	#resultLottoArr = [] //최종 로또 맞춘 결과 배열
+
+	async start() {
+		await this.inputMoneyAndLottoAmount()
+		this.randomNumberArr()
+		await this.inputWinnerNumber()
+		await this.inputBonus()
+		this.getResultLotto()
+		this.printResultLotto()
+		this.getTotalReturnPrice()
+	}
+
+	//돈 입력 및 로또 구매 수 출력
+	async inputMoneyAndLottoAmount() {
+		const userInputPrice = await MissionUtils.Console.readLineAsync(
+			MESSAGE.INPUT.PRICE_NUMBER,
+		)
+		const Validate = new validate()
```

Comment:
class 명은 대문자로 시작하고, instance는 소문자로 시작하는게 맞을 것 같아용
```suggestion
		const validate = new Validate();
```

그리구 `Validate`라는 class는 여기 `LottoGame` 클래스 내부에서 자주 사용되기 때문에 constructor로 초기화될 때 부터 인스턴스를 생성해서 써도 좋을 것 같아용

```js
constructor () {
  this.validate = new Validate();
}

// 유효성 검사 필요할 때 
this.validate.inputMoney(userInputPrice);
```

이런식으로 하면 `LottoGame`내부 어디서든 `validate` 인스턴스를 불러올 수 있고, 유효성검사가 필요할 때마다 `Validate`클래스를 초기화할 필요가 없어질 것 같습니다!

Reply:
아 그렇군요!!! 정말 감사합니당 리팩토링때 수정해야겠군요!